core/embed/rust/src/ui/geometry.rs

```diff
@@ -471,6 +475,31 @@ impl LinearLayout {
             cursor += spacing;
         }
     }
+
+    /// Arranges number of items of the same size into `area`. The `sink`
+    /// closure is called `count` times with top left point of each item as
+    /// argument. Items are centered along the cross axis.
+    pub fn arrange_uniform(
+        &self,
+        area: Rect,
+        count: usize,
+        size: Offset,
+        sink: &mut dyn FnMut(Point),
+    ) {
+        let item_size = self.axis.main(size.x, size.y);
```

Comment:
`size.axis(self.axis)`?

Reply:
Good catch. Looks funny though, i wonder if there's a better name for the method:)

Reply:
Fixed in c1588ff.